xarray/plot/facetgrid.py

```diff
@@ -314,10 +314,10 @@ def _finalize_grid(self, *axlabels):
             if namedict is None:
                 ax.set_visible(False)
 
-    def add_line_legend(self, huelabel):
+    def _add_line_legend(self, hueplt, huelabel):
```

Comment:
Do we want this method to be private? Or maybe call it just `add_legend` for consistency with `add_colorbar` and seaborn?

Reply:
I made it private because it wasn't usable without giving it data. Fixed that now.